src/core/drive/form_submission.ts

```diff
@@ -24,6 +24,20 @@ export enum FormSubmissionState {
   stopped,
 }
 
+export enum FormEnctype {
+  applicationXWWWFormUrlEncoded = "application/x-www-form-urlencoded",
+  multipartFormData             = "multipart/form-data",
+  textPlain                     = "text/plain"
+}
+
+export function formEnctypeFromString(method: string): FormEnctype {
```

Comment:
Are the `export`s for `FormEnctype` and `formEnctypeFromString` leftovers? I don't see these being used outside of here?